dassl/modeling/ops/conv.py

```diff
@@ -0,0 +1,97 @@
+"""
```

Comment:
I suggest changing `conv.py` to `conv2d_dynamic.py`

Reply:
The advantage of naming it `conv.py` is that if there are new convolutional layers in the future, they can be placed in this file. It does not only contain Conv2dDynamic, although it is currently the case.

Reply:
ok makes sense